### DISCUSSION TITLE
Populating "themes" directory doesn't add themes

### DISCUSSION THREAD

**Discussion by thomas-neuburger** [KDE/ghostwriter#811] [Q&A]
I've populated the ~/.config/ghostwriter/themes directory with theme folders (each containing a theme.cfg file), but unlike the documentation says, none of these themes shows up when I select Settings > Theme. It looks like only a <theme>.json file is presented as a theme. 

Has something changed, or am I missing a step?

Thanks.

**Comment by thomas-neuburger** [comment]
Looks like 2.0 changed the themes format. I'd be tempted to back-install 1.8 just to get the good themes back. Anyone know of a conversion tool?

**Reply by kjhambrick** [reply]
thomas-neuburger --

The [Dracula Theme](https://draculatheme.com/ghostwriter) does include Dracula.json and it works on linux ... just copy Dracula.json into ~/.config/ghostwriter/themes/

It does look nice as Light or Dark.

The format looks simple enough but it appears to be a super-set of the old Theme.cfg Files

HTH

-- kjh

![Screenshot_20221002_172328](https://user-images.githubusercontent.com/17851972/193479010-a3d1dd19-cd33-4045-9bed-378850046cf9.png)

**Reply by bluebird770** [reply]
### Can't get it to work 
For some reason, it doesn't work for me. 
Using the latest git build.
I'm using a minimal setup using `DWM` with `xinit`.

**Reply by jcnils** [reply]
I installed ghostwriter using AUR, the path for the themes for me are located in `~/.local/share/ghostwriter/themes`

**Comment by farvardin** [comment]
> I installed ghostwriter using AUR, the path for the themes for me are located in `~/.local/share/ghostwriter/themes`

that's true. I've encountered the same on Debian / Linux Mint. ``~/.config/ghostwriter/themes/`` seemed to be a more logical and sensible place, I don't understand this change, maybe it's some KDE recommendations ?
